pkg/registry/core/service/storage/rest.go

```diff
@@ -130,6 +139,53 @@ func (rs *REST) Categories() []string {
 	return []string{"all"}
 }
 
+func (rs *REST) getNetworkStorage(tenant string) (rest.Getter, error) {
+	if atomic.LoadUint32(&rs.done) == 1 {
+		if rs.networks == nil {
+			return nil, fmt.Errorf("service storage failed to identify proper network getter")
+		}
+		return rs.networks, nil
+	}
+
+	// Slow-path.
+	rs.m.Lock()
+	defer rs.m.Unlock()
+	if rs.done == 0 {
+		if rs.networks != nil {
+			defer atomic.StoreUint32(&rs.done, 1)
+			return rs.networks, nil
+		}
+
+		crStorageGetter := extensionsapiserver.GetCustomResourceStoragesGetter()
+		if crStorageGetter == nil {
+			defer atomic.StoreUint32(&rs.done, 1)
+			return nil, fmt.Errorf("api extension server not initialized properly")
```

Comment:
I would like the error message rephrase more specifically for convenience in debugging: "Failed to get the Custom Resource Storage Getter: api extension server not ....."